script/server

```diff
@@ -7,6 +7,9 @@ set -e
 
 cd "$(dirname "$0")/.."
 
+echo "==> Tearing down any old containers..."
+docker-compose down
```

Comment:
Is it worth having this in a teardown function like so:

```bash
cleanup() {
  echo "==> Tearing down any old containers..."
  docker-compose down
}
```

Which we can then use when the script exits:

```bash
trap cleanup EXIT
```

Reply:
Oo much better 👍